### PR TITLE
Delete files, and some reformatting

### PR DESCRIPTION
**Pull request by cclark-splash**
- Delete a bunch of inherited files. If we want these, we should more deliberately add them.
- Minor reformatting and refactoring

### CODE REVIEW

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,3 +0,0 @@
-# These are supported funding model platforms
```
Comment (brettrowberry) [inline .github/FUNDING.yml:1]: yes

**Comment by brettrowberry** [comment]
Most of these files seem good to keep. I'd like to think about this more.

**Comment by cclark-splash** [comment]
- If we keep them, should we add them to our other open-source repositories? 
- And if so, is this exactly the text we want?
- Do we want to maintain all of these files?

In my opinion, files like these actually create a higher barrier to entry than just not having one at all, until you have a specific need for them.

**Comment by brettrowberry** [comment]
TechHaus?

**Pull request closed without merging**